kaa/scenes.pxi

```diff
@@ -113,8 +114,14 @@ cdef class Scene:
         readonly _SceneCamera camera
 
     def __cinit__(self):
-        c_log_dynamic(CLogLevel.debug, CLogCategory.engine,
-                    "Initializing Scene")
+        if get_c_engine() == NULL:
```

Comment:
use `is_engine_initialized()` for `kaacore` since `get_engine()` throws if engine is not initialized